evm_loader/cli/src/program_options.rs

```diff
@@ -151,6 +151,15 @@ fn trx_params<'a, 'b>(cmd: &'static str, desc: &'static str) -> App<'a, 'b> {
                 .validator(is_valid_u256)
                 .help("Gas limit"),
         )
+        .arg(
+            Arg::with_name("access_list")
+                .long("access-list")
+                .takes_value(true)
+                .required(false)
+                .multiple(true)
+                // .number_of_values(1) // Access list takes at least 1 value
```

Comment:
Do we need it or not? It's the right time to make a decision.

Reply:
removed it, thanks!